### PR TITLE
New localadmin features in Belgium

### LINKED ISSUE #288: Review locality for Bergen and Schilde, Belgium
This came in through support:

"It seems your data for Belgium is off, almost every address search will return the neighborhood as locality instead of the actual locality, which in this example should be “Schilde”. So every “city” is basically wrong for our country."

image

Result from search for 49 Heidedreef, Bergen, Belgium:

"properties": {
        "id": "be/flanders:617473",
        "gid": "openaddresses:address:be/flanders:617473",
        "layer": "address",
        "source": "openaddresses",
        "name": "49 Heidedreef",
        "housenumber": "49",
        "street": "Heidedreef",
        "postalcode": "2970",
        "country": "Belgium",
        "country_gid": "whosonfirst:country:85632997",
        "country_a": "BEL",
        "macroregion": "Vlaams Gewest",
        "macroregion_gid": "whosonfirst:macroregion:404227357",
        "region": "Antwerpen",
        "region_gid": "whosonfirst:region:85681715",
        "county": "Antwerpen",
        "county_gid": "whosonfirst:county:102050005",
        "locality": "Bergen",
        "locality_gid": "whosonfirst:locality:101755783",
        "label": "49 Heidedreef, Bergen, Belgium"
      }

### PR DESCRIPTION
**Pull request by stepps00**
Fixes #288.

This PR includes 309 new `localadmin` features for the Flemmish region of Belgium. 
- Imports new `localadmin` features with:
  - Source properties ([source](https://github.com/whosonfirst/whosonfirst-sources/blob/master/sources/begov.json))
  - New `lbl:latitude`, `lbl:longitude`, `reversegeo:latitude`, and `reversegeo:longitude`fields from Mapshaper
- These features do not include any hierarchy or belongsto data; a `country_id` can be added, but the others will need to come from PIPing.

[Here](https://github.com/mapzen/whosonfirst-toolbox/commit/76280a865602e3a1f88686d1b9593594bcaa7684) are the tools used for this PR.

### CODE REVIEW

File: data/110/883/000/1/1108830001.geojson [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    "begov:oppervl":43665535.66,
+    "begov:terrid":"104",
+    "begov:uidn":"358",
+    "edtf:cessation":"uuuu",
+    "edtf:inception":"uuuu",
+    "geom:area":0.005614,
+    "geom:bbox":"4.5300874167,51.1482850793,4.64391467859,51.2361048279",
+    "geom:latitude":51.192804,
+    "geom:longitude":4.59063,
+    "iso:country":"",
```
Comment (nvkelso) [inline data/110/883/000/1/1108830001.geojson]: We should fill in `iso:country` and `wof:country` (below) as this won't be edited by PIPing. (Applies to all new features.)

File: data/110/883/000/1/1108830001.geojson [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    "mps_x":4.591489,
```
Comment (nvkelso) [inline data/110/883/000/1/1108830001.geojson]: Please set the lbl centroid to the Mapshaper values, and set the source for same to mps.

File: data/110/883/000/1/1108830001.geojson [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    "wof:created":1488241059,
+    "wof:geomhash":"25b185c455442bf5bd976794f889dc7b",
+    "wof:hierarchy":[],
+    "wof:id":1108830001,
+    "wof:lastmodified":1488241059,
+    "wof:name":"Ranst",
+    "wof:placetype":"localadmin",
+    "wof:superseded_by":[],
+    "wof:supersedes":[],
+    "wof:tags":[]
```
Comment (nvkelso) [inline data/110/883/000/1/1108830001.geojson]: How did this get here? It's not wrong, just weird.

File: data/110/883/000/1/1108830001.geojson (hunk 2 above)
Comment (nvkelso) [inline data/110/883/000/1/1108830001.geojson]: Hmm, some other records (e.g.: data/110/883/008/1/1108830081.geojson) do include the label centroid (but not the source)?

File: data/110/883/008/1/1108830081.geojson [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    "edtf:cessation":"uuuu",
+    "edtf:inception":"uuuu",
+    "geom:area":0.00017,
+    "geom:bbox":"5.41543222615,50.7180865816,5.43627110919,50.7354108017",
+    "geom:latitude":50.725941,
+    "geom:longitude":5.425431,
+    "iso:country":"",
+    "lbl:latitude":50.727313,
+    "lbl:longitude":5.425282,
+    "reversegeo:latitude":50.727313,
```
Comment (nvkelso) [inline data/110/883/008/1/1108830081.geojson]: Unless we know this to be accurate, might be better to skip adding?

File: data/110/883/000/1/1108830001.geojson [hunk 5]
```diff
@@ -0,0 +1,44 @@
+{
+  "id": 1108830001,
+  "type": "Feature",
+  "properties": {
+    "begov:datpublbs":"1982/12/29",
```
Comment (nvkelso) [inline data/110/883/000/1/1108830001.geojson]: We're importing this data from `begov` but I don't see a src for the geom saying that's where sourcing it.

File: data/110/883/008/1/1108830081.geojson (hunk 4 above)
Comment (nvkelso) [inline data/110/883/008/1/1108830081.geojson]: Oh, I see. Keep it in I guess?

File: data/110/883/000/1/1108830001.geojson (hunk 1 above)
Comment (stepps00) [inline data/110/883/000/1/1108830001.geojson]: These properties were added in commit  	13c9749.

File: data/110/883/000/1/1108830001.geojson (hunk 2 above)
Comment (stepps00) [inline data/110/883/000/1/1108830001.geojson]: This work is included in commit 13c9749.

File: data/110/883/000/1/1108830001.geojson (hunk 3 above)
Comment (stepps00) [inline data/110/883/000/1/1108830001.geojson]: That was auto-populated, likely through the `exporter` tool.

File: data/110/883/000/1/1108830001.geojson (hunk 5 above)
Comment (stepps00) [inline data/110/883/000/1/1108830001.geojson]: New `src:geom` property was added in commit 833d57a

**Comment by stepps00** [comment]
Comments in-line, this branch should contain all requested changes now. 

Over to @nvkelso for a second glance.

**Comment by nvkelso** [comment]
Other than https://github.com/whosonfirst-data/whosonfirst-data/pull/679#commitcomment-21092807, LGTM.

**Comment by stepps00** [comment]
Updated the `lbl:centroid` property to `src:lbl:centroid`, following the convention we've used in other records, in commit 8b01359.

**Comment by nvkelso** [comment]
Over to @thisisaaronland for PIPing.

**Pull request merged**